integration-tests/storybook-for-react/stories/index.js

```diff
@@ -111,6 +111,9 @@ storiesOf('addWithPercyOptions', module)
   .add('with skip option', () => <span>Will not Render</span>, {
     percy: { skip: true },
   })
+  .add('with minimum height', () => <span style={{ height: '20px' }}>Renders with multiple</span>, {
+    percy: { minimum_height: 1500 },
```

Comment:
```suggestion
    percy: { minHeight: 1500 },
```

I think `minHeight` would be more fitting here.

reference: https://github.com/percy/percy-agent/blob/master/src/percy-agent-client/snapshot-options.ts#L5